cpp/oneapi/dal/algo/dbscan/common.hpp

```diff
@@ -48,6 +49,34 @@ using v1::by_default;
 
 } // namespace method
 
+/// Represents result option flag
+/// Behaves like a regular :expr`enum`.
+class result_option_id : public result_option_id_base {
+public:
+    result_option_id() : result_option_id_base{} {}
+    result_option_id(const result_option_id_base& base) : result_option_id_base{ base } {}
+};
+
+namespace detail {
+
+ONEDAL_EXPORT result_option_id get_responses_id();
+ONEDAL_EXPORT result_option_id get_core_observation_indices_id();
+ONEDAL_EXPORT result_option_id get_core_observations_id();
+ONEDAL_EXPORT result_option_id get_core_flags_id();
+
+} // namespace detail
+
+/// Result options are used to define
+/// what should algorithm return
+namespace result_options {
+
+const result_option_id responses = detail::get_responses_id();
+const result_option_id core_observation_indices = detail::get_core_observation_indices_id();
+const result_option_id core_observations = detail::get_core_observations_id();
+const result_option_id core_flags = detail::get_core_flags_id();
```

Comment:
It's wrong! Please mark them as `const inline`

Reply:
done